sdks/python/apache_beam/coders/coders.py

```diff
@@ -220,11 +220,10 @@ def __repr__(self):
   def __eq__(self, other):
     return (self.__class__ == other.__class__
             and self._dict_without_impl() == other._dict_without_impl())
+  # pylint: enable=protected-access
```

Comment:
Do we need this pylint comment?

Reply:
It's a matching comment for the comment in line 219. I think the comment in #219 is still needed.

Reply:
Got it. L219 was not visible by default.